### ISSUE TITLE
Error in Parsing articles from pmc database in XML mode

### ISSUE THREAD

**Issue by Dhruv-Aga** [biopython/biopython#3288]
### Parse Error

I am reporting a problem with Biopython 1.78, Python 3.8.3, and Win10
the system as follows:

```python
>>> import sys; print(sys.version)
3.8.3 (tags/v3.8.3:6f8c832, May 13 2020, 22:37:02) [MSC v.1924 64 bit (AMD64)]
>>> import platform; print(platform.python_implementation()); print(platform.platform())
CPython
Windows-10-10.0.19041-SP0
>>> import Bio; print(Bio.__version__)
1.78
```
Need urgent help, is this issue for all the versions and article or it's need issue
### Expected behaviour

(*Should return Generator of the parsed XML like this [link](https://gist.github.com/Dhruv-Aga/c6f6f47cf4fd5f7a7bad78622f5a4709#file-result-py)*)

### Actual behaviour

(*Traceback (most recent call last):
  File ".\temp.issue.py", line 9, in <module>
    print(list(records))
  File "env\lib\site-packages\Bio\Entrez\Parser.py", line 354, in parse
    self.parser.Parse(data, False)
  File "C:\A\31\s\Modules\pyexpat.c", line 618, in StartNamespaceDecl
  File "env\lib\site-packages\Bio\Entrez\Parser.py", line 437, in startNamespaceDeclHandler
    assert uri == "http://www.w3.org/1998/Math/MathML"
AssertionError*)

### Steps to reproduce

`pip install biopython`
then try executing below code:
```python
from Bio import Entrez
Entrez.email = "example@gmail.c"
handle = Entrez.efetch(
    db="pmc",
    retmode="xml",
    id="PMC7279775"
)
records = Entrez.parse(handle, validate=False, escape=False)
print(list(records))
```

link of this code: [code](https://gist.github.com/Dhruv-Aga/c6f6f47cf4fd5f7a7bad78622f5a4709#file-issue-code-py)

**Comment by peterjc** [comment]
The assertion failed, the url is now http://www.w3.org/1999/xlink

I had to tweak the parser to see that, adding an assert message:

```
$ git diff
diff --git a/Bio/Entrez/Parser.py b/Bio/Entrez/Parser.py
index 77a816f93..c9843d166 100644
--- a/Bio/Entrez/Parser.py
+++ b/Bio/Entrez/Parser.py
@@ -434,8 +434,8 @@ class DataHandler:
             # to find out their first and last invocation for each namespace.
             self.namespace_level[prefix] += 1
             self.namespace_prefix[uri] = prefix
-            assert uri == "http://www.w3.org/1998/Math/MathML"
-            assert prefix == "mml"
+            assert uri == "http://www.w3.org/1998/Math/MathML", uri
+            assert prefix == "mml", prefix
 
     def endNamespaceDeclHandler(self, prefix):
         """Handle end of an XML namespace declaration."""
```

I guess this is from something new the NCBI files are doing...

**Comment by Dhruv-Aga** [comment]
> The assertion failed, the url is now http://www.w3.org/1999/xlink
> 
> I had to tweak the parser to see that, adding an assert message:
> 
> ```
> $ git diff
> diff --git a/Bio/Entrez/Parser.py b/Bio/Entrez/Parser.py
> index 77a816f93..c9843d166 100644
> --- a/Bio/Entrez/Parser.py
> +++ b/Bio/Entrez/Parser.py
> @@ -434,8 +434,8 @@ class DataHandler:
>              # to find out their first and last invocation for each namespace.
>              self.namespace_level[prefix] += 1
>              self.namespace_prefix[uri] = prefix
> -            assert uri == "http://www.w3.org/1998/Math/MathML"
> -            assert prefix == "mml"
> +            assert uri == "http://www.w3.org/1998/Math/MathML", uri
> +            assert prefix == "mml", prefix
>  
>      def endNamespaceDeclHandler(self, prefix):
>          """Handle end of an XML namespace declaration."""
> ```
> 
> I guess this is from something new the NCBI files are doing...

I did the same but after this error, another error occurs on line 650, in function `endRawElementHandler()`
at line code this, 
```uri, name = name.split()```
Error:
    uri, name = name.split()
ValueError: not enough values to unpack (expected 2, got 1)

**Comment by peterjc** [comment]
Sorry, I meant to give the unexpected URL in my comment. I've not dug into it but it looks like the NCBI have added something new (XLINK) which the parser is not handling. Getting the parser to ignore it might be pragmatic, over to @mdehoon

**Comment by mdehoon** [comment]
@Dhruv-Aga  Would you be able to provide a patch?

**Comment by Dhruv-Aga** [comment]
@mdehoon Yes sure, I can provide a patch, it is work in progress.

**Comment by mdehoon** [comment]
Fixed by #3289 .

**Comment by jpzhangvincent** [comment]
Please release the new version with this patch so we can install from PyPi! Thanks!
